modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/modelbase-header.mustache

```diff
@@ -341,7 +428,17 @@ std::shared_ptr<HttpContent> ModelBase::toHttpContent( const utility::string_t&
     content->setData( std::shared_ptr<std::istream>( new std::stringstream( utility::conversions::to_utf8string(json_array.serialize()) ) ) );
     return content;
 }
-
+template <typename T>
+std::shared_ptr<HttpContent> ModelBase::toHttpContent( const utility::string_t& name, const std::set<T>& value, const utility::string_t& contentType )
+{
+    web::json::value json_array = ModelBase::toJson(value);
+    std::shared_ptr<HttpContent> content(new HttpContent);
```

Comment:
I'd use std::make_shared here

```std::shared_ptr<HttpContent> content = std::make_shared<HttpContext>();```

Here's short explanation why:
https://stackoverflow.com/questions/18301511/stdshared-ptr-initialization-make-sharedfoo-vs-shared-ptrtnew-foo

Reply:
Sure, I'll be glad to do it!
If it's ok I also wanted to remove the nullptr check on reference variables. There are a few